src/System.DirectoryServices.Protocols/src/FxCopBaseline.AnyOS.cs

```diff
@@ -0,0 +1 @@
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Security", "CA1821", Justification = "Finalizer has implementation in Windows version.")]
```

Comment:
Wasn't sure if I should do this or remove the finalizers from the reference assembly.  /cc @terrajobst